tests/unit/conftest.py

```diff
@@ -42,6 +42,14 @@
     sqlalchemy_version >= packaging.version.parse("1.4"),
     reason="requires sqlalchemy 1.3 or lower",
 )
+sqlalchemy_before_2_0 = pytest.mark.skipif(
```

Comment:
there are multiple checks here for sqlalchemy 1.3, 1.4, 2.0+
Based on removing some of the code that requires 1.2 or 1.3, we should be able to edit this section and potentially decrease the number of checks.

Reply:
Will be removing this in #987